frontend/src/app/shared/components/enterprise-banner/enterprise-banner.component.html

```diff
@@ -7,19 +7,30 @@
 
     <img [src]="image.enterprise_edition" class="hidden-for-mobile op-enterprise-banner--image">
     <div class="op-toast--content">
-      <div class="op-enterprise-banner--title">
-        <op-icon icon-classes="button--icon icon-medal"></op-icon> &nbsp;
-        <span [textContent]="text.enterpriseFeature"></span>
+      <div class="op-enterprise-banner--header">
+        <op-icon icon-classes="button--icon icon-medal"></op-icon>
```

Comment:
always use spot icons from now on
```suggestion
        <span class="spot-icon spot-icon_medal"></span>
```